composer/content.md

```diff
@@ -91,42 +92,42 @@ Our image is aimed at quickly running Composer without the need for having a PHP
 
 Suggestions:
 
--	use [`--ignore-platform-reqs`](https://getcomposer.org/doc/03-cli.md#install-i):
-
-	```console
-	$ composer install --ignore-platform-reqs
-	```
-
--	specify the target [platform](https://getcomposer.org/doc/06-config.md#platform) in your `composer.json`:
+-	(recommended) specify the target [platform](https://getcomposer.org/doc/06-config.md#platform) in your `composer.json`:
 
 	```json
 	{
 	  "config": {
 	    "platform": {
-	      "php": "7.1.3"
+	      "php": "MAJOR.MINOR.PATCH"
 	    }
 	  }
 	}
 	```
 
+-	use [`--ignore-platform-reqs`](https://getcomposer.org/doc/03-cli.md#install-i):
+
+	```console
+	$ composer install --ignore-platform-reqs
+	```
+
 ### PHP extensions
 
 We aim to deliver an image that is as lean as possible, built for running Composer only. Sometimes dependencies or Composer [scripts](https://getcomposer.org/doc/articles/scripts.md) require the availability of certain PHP extensions.
 
 Suggestions:
 
+-	(recommended) create your own build image and [install](https://getcomposer.org/doc/faqs/how-to-install-composer-programmatically.md) Composer inside it.
+
+	**Note:** Docker 17.05 introduced [multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/), simplifying this enormously:
+
+	```dockerfile
+	COPY --from=%%IMAGE%% /usr/bin/composer /usr/bin/composer
+	```
+
```

Comment:
`config.platform` should work for extensions as well. Maybe we can add this to the list of suggestions?